### DISCUSSION TITLE
How to update azure repos from github?

### DISCUSSION THREAD

**Discussion by wangjj2019** [community/community#21567] [New to GitHub]
<p>We have a project now. In the customer side, we use azure repos + azure Devops on the intranet, and we use GitHub for development. We can’t access the customer’s repos code base outside. In this case, is it possible for us to develop and upload code on GitHub and let customers pull code from GitHub to update azurerepos?</p>

**Comment by sr229** [comment, marked as answer]
<p>Yes, it’s possible to do this using GitHub Actions. You may do a one-way synchronization by pushing to the DevOps repo. If you want a more advanced way, you can first do a synchronization of the DevOps tree, check against the GitHub copy, merge them together, and push it on both repos.</p>
<p>I do recommend to use GitHub Private repos instead for this sort of thing but I assume there’s some security requirements on the customer, so this is the best thing you will have at the moment, aside from using GHES in place of Azure Repos.</p>
